src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -509,6 +509,7 @@
   "HardlinkCopyFiles": "Hardlink/Copy Files",
   "HasMissingSeason": "Has Missing Season",
   "Health": "Health",
+  "HealthMessagesInfoBox": "You can find more information about the cause of these health check messages by clicking the wiki link (book icon) at the end of the row, or by checking your [logs]({link}). If you have difficulty interpreting these messages then you can reach out to our support, at the links below.",
```

Comment:
Would you hate me for suggesting replacing (book icon) with the actual icon?

Reply:
wouldn't this only work by passing an Icon component as a token into translate.js - which it doesn't accept right now?

Reply:
Yeah, it'd require some additional work to support injecting an icon like we do for other cases, something we could do in a follow up PR though.

Reply:
> in a follow up PR though.

upvote :D